automation/taskcluster/decision_task.py

```diff
@@ -113,9 +123,39 @@ def create_variant_test_task(variant):
             'symbol': 'T',
             'tier': 1,
         },
+        artifacts=_craft_artifacts_from_variant(variant),
     )
 
 def _craft_treeherder_platform_from_variant(variant):
+    architecture, build_type = _get_architecture_and_build_type_from_variant(variant)
+    return 'android-{}-{}'.format(architecture, build_type)
+
+
+def _craft_artifacts_from_variant(variant):
+    return {
+        'public/target.apk': {
+            'type': 'file',
+            'path': _craft_apk_full_path_from_variant(variant),
+            'expires': taskcluster.stringDate(taskcluster.fromNow(lib.tasks.DEFAULT_EXPIRES_IN)),
+        }
+    }
+
+
+def _craft_apk_full_path_from_variant(variant):
+    architecture, build_type = _get_architecture_and_build_type_from_variant(variant)
+
+    short_variant = variant[:-len(build_type)]
+    shorter_variant = short_variant[:-len(architecture)]
+
+    return '/build/reference-browser/app/build/outputs/apk/{short_variant}/{build_type}/app-{shorter_variant}-{architecture}-{build_type}-unsigned.apk'.format(
```

Comment:
Nit: I wonder if we should use a constant file here as well.  Do we expect this to change at some point in the future?

Reply:
I don't know. So far, this string is defined once. I'm fine extracting it in a constant if it becomes used elsewhere